apps/client/src/client.module.ts

```diff
@@ -18,8 +18,8 @@ import { CommonModule } from './common/common.module';
 import { RequestContextModule } from '@libs/request-context';
 import { CoreModule } from '@libs/core';
 import eventEmitterConfig from '../../../config/event-emitter.config';
-import { PusherModule } from '@libs/pusher';
-import firebaseConfig from '../../../config/firebase.config';
+//import { PusherModule } from '@libs/pusher';
```

Comment:
If we don't use it, it is better to remove it. You can add README.md with an example of how to use it in the `@libs/pusher`